cmd/kvdpa-cli/kvdpa-cli.go

```diff
@@ -72,6 +73,19 @@ func getAction(c *cli.Context) error {
 	return nil
 }
 
+func addAction(c *cli.Context) error {
+	if c.Args().Len() != 2 {
+		usage := fmt.Sprintf("Usage: %v %v %v \n", c.App.Name, c.Command.Name, c.Command.ArgsUsage)
+		return errors.New(usage)
+	}
+
+	mgmtDevName := c.Args().Get(0)
+	devName := c.Args().Get(1)
+
+	_, err := vdpa.AddVdpaDevice(mgmtDevName, devName)
```

Comment:
Adding this commit before actually adding `AddVdpaDevice` means this commit does not compile. It would break git bisects or other experiments. It would be much better to sort commits in another way. First adding functionality, then adding clients that use it.

Reply:
Rebasing again from master fixes the problem